tests/end-to-end/upgrade/pkg/tests/eventmesh-migration/helpers/serviceinstance.go

```diff
@@ -0,0 +1,28 @@
+package helpers
+
+import (
+	"github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1beta1"
```

Comment:
Please make the alias more specific, e.g. `scv1beta1`

Reply:
done